actions/src/create-cut-pr/main.ts

```diff
@@ -0,0 +1,176 @@
+import { branch, changelog, type Ctx } from "@/lib";
+import type { PullRequest } from "@/lib/github";
+import { CHANGELOG_LABEL, MAJOR_SEVERITY_LABEL } from "@/lib/labels";
+import { setPackageVersion } from "@/lib/package";
+import { severityFits, type Severity } from "@/lib/severity";
+
+/**
+ * Determine what the next cut version will be, based existing release/x.y` branches in the repo.
```

Comment:
is the tick (`) intended?
```suggestion
 * Determine what the next cut version will be, based on existing release/x.y branches in the repo.
```